pkgs/applications/audio/quodlibet/default.nix

```diff
@@ -35,11 +35,11 @@ python3.pkgs.buildPythonApplication rec {
 
   checkPhase = ''
     runHook preCheck
-    env XDG_DATA_DIRS="$out/share:${gtk3}/share/gsettings-schemas/${gtk3.name}:$XDG_DATA_DIRS" \
+    env XDG_DATA_DIRS="$out/share:${gtk3}/share/gsettings-schemas/${gtk3.name}:$XDG_ICON_DIRS:$XDG_DATA_DIRS" \
       HOME=$(mktemp -d) \
       xvfb-run -s '-screen 0 800x600x24' dbus-run-session \
         --config-file=${dbus.daemon}/share/dbus-1/session.conf \
-        py.test${stdenv.lib.optionalString (xineBackend || !withGstPlugins) " --ignore=tests/plugin/test_replaygain.py"}
+        py.test ${stdenv.lib.optionalString (xineBackend || !withGstPlugins) " --ignore=tests/plugin/test_replaygain.py"} --ignore=tests/quality
```

Comment:
Does `--no-quality` suggested in the other PR not work?

Reply:
No, that is part of the testing script from `quodlibet` that this is not running.  It seems that this build script runs the tests directly instead and errored with `--no-quality`.

Here is where it is defined in quodlibet:

https://github.com/quodlibet/quodlibet/blob/30293621f466cbc0076ed2db703702ef0234a853/quodlibet/gdist/tests.py#L42